### PR TITLE
[FEATURE REQ] bitcoin-cli man page + gpg verification improvement

### PR DESCRIPTION
**Pull request by st3b1t**
check signs count

#### What

the batch output of gpg verify leads to misunderstandings

#### Why

simplify the verification of good signs

#### How

one line of bash

### CODE REVIEW

**Comment by st3b1t** [comment]
seconda commit contains installations of man pages

**Review by twofaktor — APPROVED** [review]
Good idea, tACK, but what do you mean by "the batch output of gpg verify leads to misunderstandings"?

File: guide/bitcoin/bitcoin-client.md [hunk 1]
```diff
@@ -120,6 +120,12 @@ Expected output:
   > Primary key fingerprint: ...
   ```
 
+* This command must generate a number greater than 0
+
+  ```sh
+  $ gpg --verify SHA256SUMS.asc 2>&1 | grep Good | wc -l
```
Comment (twofaktor) [inline guide/bitcoin/bitcoin-client.md:126]: Why this is useful? I got number 12 nothing more, what does mean this?

**Review by twofaktor — APPROVED** [review]

**Comment by twofaktor** [comment]
Let's go to the main! 🚀

File: guide/bitcoin/bitcoin-client.md (hunk 1 above)
Comment (st3b1t) [reply guide/bitcoin/bitcoin-client.md]: > Good idea, tACK man page, but what do you mean by "the batch output of gpg verify leads to misunderstandings"?

this is not a problem of this great guide that guide the user through each steps.. very nice.

Unfortunately for newbies the gpg output is a hodgepodge of "WARNING, Good, Sign, emails" confusing words mixed into the same output...

**Pull request merged**